lib/stripe_mock/instance.rb

```diff
@@ -249,28 +249,6 @@ def to_faraday_hash(hash)
       response.new(hash)
     end
 
-    def calculate_fees(params)
```

Comment:
wasn't this used? I think I added this 😆 , also it should break a hint spec

Reply:
it still used :) but the method was defined twice. I guess I kept both when rebuking with master on my last commit. both methods were pretty similar anyways

Reply:
ohhhh, sounds god